legend-depot-store-api/src/main/java/org/finos/legend/depot/store/admin/api/metrics/QueryMetricsStore.java

```diff
@@ -27,7 +27,7 @@
 
     List<VersionQueryMetric> getAll();
 
-    void record(String groupId, String artifactId, String versionId);
+    void insert(VersionQueryMetric versionQueryMetric);
```

Comment:
and as per previous comment lets move any metrics related to metrics mocule, including this